beeai/supervisor/main.py

```diff
@@ -119,18 +128,21 @@ def execute(repeat: bool = typer.Option(True)):
 
 
 async def do_process_issue(key: str):
+    assert app_state.dry_run
     issue = get_issue(key, full=True)
-    result = await run_issue_workflow(issue)
+    result = await IssueWorkflow(issue, dry_run=app_state.dry_run).run()
     logger.info(
         "Issue %s processed, status=%s, reschedule_in=%s",
         key,
         result.status,
-        result.reschedule_in if result.reschedule_in > 0 else "never",
+        result.reschedule_in if result.reschedule_in >= 0 else "never",
```

Comment:
unrelated bug fix that should probably be squashed into earlier commits

Reply:
Done